glib/src/utils.rs

```diff
@@ -337,86 +114,65 @@ pub fn uri_unescape_segment(
     }
 }
 
-#[cfg(test)]
```

Comment:
The tests should all stay. The functions are still there, just in a different place.